index.php

```diff
@@ -28,6 +28,52 @@
 
 $id = required_param('id', PARAM_INT);
 
-require_login();
+$course = $DB->get_record('course', ['id' => $id], '*', MUST_EXIST);
+require_course_login($course);
```

Comment:
Who should be able to see this? Only teacher? There's a capability check missing for this site then.

Reply:
The content is for all users of the course, then require_course_login($course); should be sufficient?